sdk/consumption/Azure.ResourceManager.Consumption/src/autorest.md

```diff
@@ -88,6 +85,13 @@ rename-rules:
   URI: Uri
   Etag: ETag|etag
 
+models-to-treat-empty-string-as-null:
+  - ConsumptionLegacyReservationRecommendation
+  - ConsumptionLegacyUsageDetail
+  - ConsumptionMarketplace
+  - ConsumptionModernReservationRecommendation
+  - ConsumptionModernUsageDetail
```

Comment:
How about `PriceSheetProperties`? It also contains `MeterId`

Reply:
resolved